index.html

```diff
@@ -3,177 +3,67 @@
             <button class="no-btn" onmouseover="crazyButton()" id="btn">No</button>
         </div>
-        <div id="celebrate" style="display: none">
-            <img style="position: absolute;left: 0; top:15%; z-index: -2" src="https://media.tenor.com/qg8K8VOmzJwAAAAi/party-popper-confetti.gif" alt="celebrate"/>
-            <img style="position: absolute;right: 0; top:15%; transform: scaleX(-1); z-index: -2" src="https://media.tenor.com/qg8K8VOmzJwAAAAi/party-popper-confetti.gif" alt="celebrate"/>
-            <img style="position: absolute;right: 0; bottom:0;z-index: -2" src="https://media.tenor.com/DXL84KB6lscAAAAi/confetti-cute.gif" alt="celebrate"/>
-            <img style="position: absolute;left: 0; bottom:0;z-index: -2; transform: scaleX(-1);" src="https://media.tenor.com/DXL84KB6lscAAAAi/confetti-cute.gif" alt="celebrate"/>
+        <div class="display-none celebrate">
+            <img src="https://media.tenor.com/qg8K8VOmzJwAAAAi/party-popper-confetti.gif" alt="celebrate"/>
+            <img src="https://media.tenor.com/DXL84KB6lscAAAAi/confetti-cute.gif" alt="celebrate"/>
         </div>
-        <b style="color:white;font-family: sans-serif">-Sincerely, Eshan</b>
+       <h3 class="color-white">-Future Boyfriend, Eshan</h3>
     </div>
     <!--DIALOG CONTENT-->
     <dialog id="showDialog">
-        <div style="border: 5px double #DA291C;border-radius: 4px; padding:0 24px 24px">
-            <div style="display: flex;justify-content: end">
-                <p class="closeicon" onclick="closeDialog()" style="cursor:pointer; font-size: 30px; text-align: right; display: inline-block">X</p>
+            <div class="dialog-title">
+                <h1>YES?!</h1><br>
+                <p class="closeicon" onclick="closeDialog()">X</p>
             </div>
-            <div>
-                <h1 style="text-align: center;font-size: 55px"><span style="color: red">YES?!</span> </br>Love you & see u soon</h1>
-                <img src="https://media.tenor.com/uPqAeCeFDkkAAAAi/%E5%B9%B8%E7%A6%8F%E7%88%B1%E5%BF%83-happy.gif" alt="celebrate"/>
-                <img src="https://media.tenor.com/dLyUHPL976UAAAAi/love-in-love.gif" alt="celebrate"/>
-            </div>
-        </div>
-    </dialog>
-
-</body>
-<script>
-    let btn = document.getElementById('btn');
-    let dialog = document.getElementById('showDialog');
-    btn.style.position = "absolute";
-
-    function crazyButton() {
-        btn.style.left= (Math.random()*90)+"%";
-        btn.style.top= (Math.random()*90)+"%";
-    }
-    function closeDialog() {
-        dialog.close();
-    }
-    function showConfetti(){
-        dialog.showModal();
-        document.getElementById('confetti-wrapper').style.display = "block";
-        document.getElementById('celebrate').style.display = "block";
-    }
 
+        <div class="text-center">
+            <img src="https://media.tenor.com/uPqAeCeFDkkAAAAi/%E5%B9%B8%E7%A6%8F%E7%88%B1%E5%BF%83-happy.gif" alt="celebrate"/>
```

Comment:
#suggestion A particular gif has got empty space on the top,  need another gif

Reply:
Empty?

Reply:
[28d2af07-6eb6-4d90-bd4d-db1b77dd3bc1.webm](https://user-images.githubusercontent.com/94899684/210973665-fbb13f3f-52e3-4265-992d-cfd282bd3feb.webm)

Reply:
Theres 2 gif here actually

![image](https://user-images.githubusercontent.com/87816481/210974840-7b63194e-269d-449e-bf19-5bc13505f836.png)

Reply:
Thanks will check it out